packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -74,6 +74,7 @@
   "experimental_functions_install_dependencies": "Install dependencies from package.json after creating a function",
   "force_function_start": "SFDX: Start Function",
   "force_function_invoke": "SFDX: Invoke Function",
+  "force_function_debug_invoke": "SFDX: Debug Invoke Function",
```

Comment:
@danyusf @jag-sfdc - I'm leaning toward "Debug Invoke**d** Function" (with a "d") - and a question: can we debug non-invoked functions? If we can't and can ONLY debug invoked functions, do we just want to say Debug Function? I'll defer to Dan and his knowledge on this one.

Reply:
@jag-sfdc @ekapner This command name won't surface in UI (not available in command palette). I'm fine if we change it to "Debug Function" though.

Reply:
I'm not entirely sure I understand what force_function_debug_invoke actually does. One typical workflow for local functions development is:
- write the code
- start the function locally
- attach VSCode debugger to the local running function container, set breakpoints
- invoke the function with a test payload, hit breakpoints in VSCode, debug

Does force_function_debug_invoke basically do the attach to a locally running function started via force_function_start? Or is it doing something different? If the former, I'd say "Debug Function" makes more sense to me than "Debug Invoked Function" (since we haven't invoked it yet).

Reply:
So "Debug Send Request" is doing the both for you: (See the gif)
- attach VSCode debugger to the local running function container
- invoke the function with a test payload, hit breakpoints in VSCode, debug

This name here doesn't actually matter. The message we show is ["Debug Send Request"](https://github.com/forcedotcom/salesforcedx-vscode/pull/2606/files/caab6891692997899c9e4e3d88ee21f190cd774e#diff-7b947f8356f256896c623e6c8ef40379R164).

Reply:
Ah, I see. If the name was being shown to the user, I'd lobby for something like "Attach Debugger and Invoke Function", but since it's not really shown, "Debug Function" seems fine to me.